aws-crt/spec/crt_spec.rb

```diff
@@ -4,14 +4,14 @@
   describe '#call' do
     it 'raises an error when called on a function that returns an int' do
       expect do
-        Aws::Crt.call { Aws::Crt::Native.test_error }
+        Aws::Crt.call { Aws::Crt::Native.test_error(3) }
```

Comment:
nice. I like it!

Reply:
I'd really like to expose the enums defined in https://github.com/awslabs/aws-c-common/blob/78dbfab579dea968a116a4e97f4af04b94e0d2b5/include/aws/common/error.h#L143  - but I couldn't find an easy way - I suspect we'd need to codegen that....